### ISSUE TITLE
Make revision keyword configurable

### ISSUE THREAD

**Issue by pashkinmv** [mojohaus/flatten-maven-plugin#128]
Make next properties configurable:
1. revisionKeyword
2. sha1Keyword
3. changelistKeyword

**Comment by jeacott1** [comment]
yes please. in large multimodule projects that depend on other large multimodule projects it becomes very unclear very quickly when everything is loaded into an ide what project belongs to what.
this would help. as would just allowing a custom set of properties to be resolved, and so allowing any variation of properties to work instead of just these magic 3.

**Comment by lasselindqvist** [comment]
Is the intention here same as in issue https://github.com/mojohaus/flatten-maven-plugin/issues/190 ?

**Comment by pashkinmv** [comment]
I don't understand the intention of #190.

**Comment by lasselindqvist** [comment]
I don't think I understand what you want then. Maybe you could provide an example input and output you wish to achieve?
